ocaml/src/maintenance.ml

```diff
@@ -179,6 +179,14 @@ class client ?(retry_timeout = 60.)
          Lwt.return_unit)
         retry_timeout
 
+    method _is_cache_namespace ~namespace_id =
+      nsm_host_access # get_namespace_info ~namespace_id >>= fun (namespace, namespace_info, _, _) ->
+      let open Maintenance_config in
+      Hashtbl.exists
```

Comment:
This confused me (is this a function in Hashtbl?) until I saw you added it in the overload. 
It's ok, but may we want another name to indicate that it scans the hashtbl.